doc/JSON_FLAGS.md

```diff
@@ -4,167 +4,398 @@
 
 
 ## TODO
 
-- `Ammo type` table is very old and doesn't include many new ammo types. Consider updating it or removing altogether, as ammo types ain't no json flags at all.
-
-
-## Ammo
-
-### Ammo type
-
-These are handled through `ammo_types.json`.  You can tag a weapon with these to have it chamber existing ammo, or make your own ammo there.  The first column in this list is the tag's "id", the internal identifier DDA uses to track the tag, and the second is a brief description of the ammo tagged.  Use the id to search for ammo listings, as ids are constant throughout DDA's code.  Happy chambering!  :-)
-
-- ```120mm``` 120mm HEAT
-- ```12mm``` 12mm
-- ```20x66mm``` 20x66mm Shot (and relatives)
-- ```223``` .223 Remington (and 5.56 NATO)
-- ```22``` .22LR (and relatives)
-- ```3006``` 30.06
-- ```300``` .300 WinMag
-- ```308``` .308 Winchester (and relatives)
-- ```32``` .32 ACP
-- ```36paper``` .36 cap & ball
-- ```38``` .38 Special
-- ```40``` 10mm
-- ```40mm``` 40mm Grenade
-- ```44``` .44 Magnum
-- ```44paper``` .44 cap & ball
-- ```454``` .454 Casull
-- ```45``` .45 ACP (and relatives)
-- ```46``` 46mm
-- ```500``` .500 Magnum
-- ```50``` .50 BMG
-- ```57``` 57mm
-- ```5x50``` 5x50 Dart
-- ```66mm``` 66mm HEAT
-- ```700nx``` .700 Nitro Express
-- ```762R``` 7.62x54mm
-- ```762``` 7.62x39mm
-- ```762x25``` 7.62x25mm
-- ```84x246mm``` 84x246mm HE
-- ```8x40mm``` 8mm Caseless
-- ```9mm``` 9x19mm Luger (and relatives)
-- ```9x18``` 9x18mm
-- ```BB``` BB
-- ```RPG-7``` RPG-7
-- ```UPS``` UPS charges
-- ```ammo_flintlock``` Flintlock ammo
-- ```ampoule``` Ampoule
-- ```arrow``` Arrow
-- ```battery``` Battery
-- ```blunderbuss``` Blunderbuss
-- ```bolt``` Bolt
-- ```charcoal``` Charcoal
-- ```components``` Components
-- ```dart``` Dart
-- ```diesel``` Diesel
-- ```fish_bait``` Fish bait
-- ```fishspear``` Speargun spear
-- ```fusion``` Laser Pack
-- ```gasoline``` Gasoline
-- ```homebrew_rocket``` homebrew rocket
-- ```lamp_oil``` Lamp oil
-- ```laser_capacitor``` Charge
-- ```m235``` M235 TPA (66mm Incendiary Rocket)
-- ```metal_rail``` Rebar Rail
-- ```money``` Cents
-- ```muscle``` Muscle
-- ```nail``` Nail
-- ```pebble``` Pebble
-- ```plasma``` Plasma
-- ```plutonium``` Plutonium Cell
-- ```rebreather_filter``` Rebreather filter
-- ```shot``` Shotshell
-- ```signal_flare``` Signal Flare
-- ```tape``` Duct tape
-- ```thread``` Thread
-- ```thrown``` Thrown
-- ```unfinished_char``` Semi-charred fuel
-- ```water``` Water
-- ```paper``` Paper
-
-### Ammo effects
+- Some ammo effects (`LASER`, `NEVER_MISFIRES`) can be set as ammo `"effects"`, ammo `"flags"` or as the gun's `"ammo_effects"`.  This could use some clarification.
+- Move the Use action subsection from Comestibles to JSON_INFO.md, as those are not flags.
+- Move the Parameters sub-subsection from Faults to JSON_INFO.md, as those are not flags.
+- Move the Categories subsection from Recipes to JSON_INFO.md, as those are not flags.
```

Comment:
I don't think the documentaiton is the place to leave TODOs like this

Reply:
Some old docs have TODO sections. In the case of the current JSON_FLAGS.md, it's asking for a review or the removal of the `Ammo type` table, which I removed in the PR as ammo types are fully .jsonized (and thus aren't flags).

However, I didn't consider correct to remove the three mentioned subsections right away, as those contain information that belong to a different doc, and may require a different PR for that doc altogether.